src/components/artwork/fillwidth.tsx

```diff
@@ -63,9 +63,7 @@ export class Fillwidth extends React.Component<Props, null> {
     let dir = this.widthDiff(dimensions) < 0 ? -1 : 1
 
     // Keep looping until we get an acceptable width difference
-    const list = new Array(999)
-
-    for (let i of list) { // eslint-disable-line no-unused-vars
+    while (true) {
```

Comment:
Even better! I wasn’t sure what the rules were for nested loops (i.e. the `for` inside this `while`) 👍